Cargo.lock

```diff
@@ -2620,10 +2703,10 @@ checksum = "0434fabdd2c15e0aab768ca31d5b7b333717f03cf02037d5a0a3ff3c278ed67f"
 dependencies = [
  "libc",
  "log",
- "ndk",
+ "ndk 0.7.0",
```

Comment:
The diff-context suggests that this is `ndk-glue`, which is long gone and replaced with `android-activity`. Stale dependency?

Reply:
Should be fixed in https://github.com/ndarilek/tts-rs/pull/50.